__DEFINES/game.dm

```diff
@@ -0,0 +1 @@
+#define EMBED_THROWING_SPEED 20
```

Comment:
There IS a distinct lack of an appropriately named _DEFINE to put shit like this in, though